docs/src/snforge-advanced-features/fork-testing.md

```diff
@@ -53,7 +56,7 @@ the same fork in tests.
 [[tool.snforge.fork]]
 name = "SOME_NAME"
 url = "http://your.rpc.url"
-block_id.tag = "Latest"
+block_id.tag = "latest"
```

Comment:
It's a breaking change isn't it?

Reply:
We need to have it in the changelog if it's breaking

Reply:
Yep it is, added